test/api/layout/execute_layout_test.cpp

```diff
@@ -323,65 +323,55 @@ TEST(execute_test, many_ubs)
                                       "@}\n"
                                       "@HIBF_CONFIG_END\n"
                                       "#TOP_LEVEL_IBF fullest_technical_bin_idx:63\n"
-                                      "#LOWER_LEVEL_IBF_0 fullest_technical_bin_idx:24\n"
-                                      "#LOWER_LEVEL_IBF_1 fullest_technical_bin_idx:24\n"
-                                      "#LOWER_LEVEL_IBF_2 fullest_technical_bin_idx:24\n"
-                                      "#LOWER_LEVEL_IBF_3 fullest_technical_bin_idx:26\n"
-                                      "#LOWER_LEVEL_IBF_4 fullest_technical_bin_idx:22\n"
-                                      "#LOWER_LEVEL_IBF_5 fullest_technical_bin_idx:22\n"
-                                      "#LOWER_LEVEL_IBF_6 fullest_technical_bin_idx:22\n"
-                                      "#LOWER_LEVEL_IBF_7 fullest_technical_bin_idx:22\n"
-                                      "#LOWER_LEVEL_IBF_8 fullest_technical_bin_idx:22\n"
-                                      "#LOWER_LEVEL_IBF_9 fullest_technical_bin_idx:0\n"
-                                      "#LOWER_LEVEL_IBF_10 fullest_technical_bin_idx:0\n"
-                                      "#LOWER_LEVEL_IBF_11 fullest_technical_bin_idx:0\n"
-                                      "#LOWER_LEVEL_IBF_12 fullest_technical_bin_idx:33\n"
+                                      "#LOWER_LEVEL_IBF_0 fullest_technical_bin_idx:49\n"
+                                      "#LOWER_LEVEL_IBF_1 fullest_technical_bin_idx:18\n"
+                                      "#LOWER_LEVEL_IBF_2 fullest_technical_bin_idx:0\n"
```

Comment:
Now there are less merged bins because more user bins can be combined into the same bin since it is downscaled (relaxed fpr).